Makefile

```diff
@@ -45,7 +45,7 @@ GO_LDFLAGS=-ldflags '-s -w -X $(PKG)/version.Version=$(VERSION) -X $(PKG)/versio
 SHIM_GO_LDFLAGS=-ldflags '-s -w -X $(PKG)/version.Version=$(VERSION) -X $(PKG)/version.Revision=$(REVISION) -X $(PKG)/version.Package=$(PKG) -extldflags "-static"'
 
 TESTFLAGS_RACE=
-GO_GCFLAGS=
+GO_GCFLAGS= -gcflags=-trimpath=${GOPATH}/src
```

Comment:
`$GOPATH`, like `$PATH` can have multiple paths separated by `:`. This needs to be handled by either adding a trim for each path or just using the first path.